.travis.yml

```diff
@@ -42,6 +42,8 @@ env:
     - RUN_E2E_TESTS_USERS=true
     # E2E tests for mobile.
     - RUN_E2E_TESTS_MOBILE=true
+    # Third party folder size check
+    - RUN_THIRD_PARTY_SIZE_CHECK=true
```

Comment:
I would prefer that we collapse this into the lint check since this isn't exactly a substantially large test that it would need to stand on its own. Since there could be a possibility that Travis limits us on the number of containers we can use at any time and because this isn't a substantial test, I don't think it's a good idea to make this stand on its own.

Reply:
I moved the check to run_presubmit_checks.sh, its done in https://github.com/oppia/oppia/pull/5743/commits/69a1941d73001c4cacfc4f68c7c22b73acf8f61a